php/ext/google/protobuf/wkt.inc

```diff
@@ -319,6 +319,13 @@ static PHP_METHOD(google_protobuf_Api, setSourceContext) {
   RETURN_COPY(getThis());
 }
 
+static PHP_METHOD(google_protobuf_Api, hasSourceContext) {
```

Comment:
`php/ext/google/protobuf/wkt.inc` was generated with `bazel build //php:gen_wkt_sources`.

Unsure why this file is in git..

Reply:
This will be automatically updated after the PR is submitted, so you could leave it out of the PR if you wanted. But there's also no harm in updating it eagerly, as you have.